_posts/2023-07-20-release-2021.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  "Firmware 2021 - letzte mit 8MB support"
+author: Geno
+date:   2023-07-21 20:08:00 +0100
+---
+
+Nachdem die Testing-Firmware [2021.1.2+bremen1](https://wiki.bremen.freifunk.net/Firmware/Changelog.md#2021-1-2-bremen1) nun seit acht Monate ohne schwerwigende Probleme im Einsatz ist,
+ diese Version zur Stable-Firmware zu befördern.
+
+Diese Version ist wichtig für zukünftige Updates und ist daher zwingend notwendig, bevor wir vielleicht auf [2023.1](https://wiki.ffhb.de/Firmware/Changelog.md#2023-1) umstellen.
+
+Doch viel wichtiger, die noch ca. 46 Stück des TP-Link WR841 bekommen hiermit letztes Update.
+Wir haben zwar bereits in [Zukunft des TP-Link WR841](/blog/2020/07/21/zukunft-841.html) im Jahre 2020 davor gewarnt.
```

Comment:
Wir haben bereits in [Zukunft des TP-Link WR841](/blog/2020/07/21/zukunft-841.html) im Jahre 2020 davor gewarnt, nun ist es so weit: Die 4MB Flash und 32MB RAM Modelle haben ab jetzt quasi ausgedient. Damit bekommen die ~46 verbliebenen TP-Link WR841 hiermit ihr letztes Update.